build.py

```diff
@@ -15,7 +15,7 @@
 
 if Configuration.current.target.sdk == OSType.Linux:
 	foundation.CFLAGS = '-DDEPLOYMENT_TARGET_LINUX -D_GNU_SOURCE '
-	foundation.LDFLAGS = '${SWIFT_USE_LINKER} -Wl,@./CoreFoundation/linux.ld -lswiftGlibc `icu-config --ldflags` -Wl,-defsym,__CFConstantStringClassReference=_TMC10Foundation19_NSCFConstantString -Wl,-Bsymbolic '
+	foundation.LDFLAGS = '${SWIFT_USE_LINKER} -Wl,@./CoreFoundation/linux.ld -lswiftGlibc `pkg-config icu-uc icu-i18n --libs` -Wl,-defsym,__CFConstantStringClassReference=_TMC10Foundation19_NSCFConstantString -Wl,-Bsymbolic '
```

Comment:
The tool itself reads a file.  However, the path in which the tool searches is platform dependent as the libraries (e.g. on ELF) would be different.  So, you end up linking against the libraries for the wrong architecture if you do not use the right path (which is hard) or use the target triple prefixed version.  The reason that the version you looked at was symlinked is because the unprefixed version was for the target that you were targeting.  Set up a full image for a foreign architecture and you will find it to be more than a symlink.